portfolio/src/main/webapp/images.html

```diff
@@ -229,6 +228,9 @@ <h2>June 2018-Present</h2>
           </a>
         </div>
       </div>
+      <div class="buttons-row">
+        <button class="button"><a href="index.html">Back</a></button>
```

Comment:
No action required: I'm pretty sure that nesting an anchor tag inside a button or a button inside an anchor is not valid HTML, but after a little bit of searching, I haven't found any non-tedious way to make a link that looks like a button, so if the browsers handle this fine, I'm fine with it.